build/commands/lib/util.js

```diff
@@ -171,7 +171,7 @@ const util = {
           "src/chrome/tools/test/reference_build/chrome_win": "%None%"
         },
         custom_vars: {
-          "checkout_pgo_profiles": "%False%"
+          "checkout_pgo_profiles": config.chromePgoPhase === 2 ? "%True%" : "%False%"
```

Comment:
To make this works you need to run `npm run sync -- --init` to regenerate `.gclient`.
IMHO it's ok for CI (because it always runs `sync -- init`).

Reply:
CI will soon move away from `init` and just do `sync`.

Reply:
So your suggest to regenerate .gclient on demand?

Reply:
Why do we need to regenerate?

Reply:
In case you change chrome_pgo_phase in config.
Now you have to run sync --init, it's not obviously.

Reply:
@atuchin-m `chrome_pgo_phase` has nothing to do with `.gclient`

Reply:
I think this should regenerate automatically if `chromePgoPhase == 2`

Reply:
@mihaiplesa we still need to use `sync -- init` the first time because that is what generates the `.gclient` file in the first place and we don't want it to always generate on `sync` because then ios and android (and CI) would always have to specify the target os when running `sync`

Reply:
@atuchin-m maybe the best thing here is to warn if `chrome_pgo_phase` has changed, but I need to think about how we want to handle this. 
cc @petemill because I'd also like to switch to only updating args.gn on `sync` so you don't have to include all the args with each command (build, create_dist, etc...)

Reply:
There is a warning in the chromium code with suggesting to set `checkout_pgo_profiles: True` manually.
IMHO it's enough
``
RuntimeError: requested profile "E:\work\brave-browser\src\chrome\build\pgo_profiles\chrome-win64-4692-1638292939-5a7a7c95dd44988573bb4d4b0e11c8bb26816e80.profdata" doesn't exist, please make sure "checkout_pgo_profiles" is set to True in the "custom_vars" section of your .gclient file, e.g.:
solutions = [
  {
    "name": "src",
    # ...
    "custom_vars": {
      "checkout_pgo_profiles": True,
    },
  },
],
and then run "gclient runhooks" to download it. You can also simply disable the PGO optimizations by setting |chrome_pgo_phase = 0| in your GN arguments.
ERROR at //build/config/compiler/pgo/BUILD.gn:59:35: Script returned non-zero exit code.
      pgo_data_path = rebase_path(exec_script("//tools/update_pgo_profiles.py",
                                  ^----------
Current dir: E:/work/brave-browser/src/out/Release/
Command: E:/work/brave-browser/src/brave/vendor/depot_tools/bootstrap-2@3_8_10_chromium_23_bin/python3/bin/python3.exe E:/work/brave-browser/src/tools/update_pgo_profiles.py --target win64 get_profile_path
Returned 1.
See //build/config/BUILDCONFIG.gn:347:3: which caused the file to be included.
  "//build/config/compiler/pgo:default_pgo_flags",
  ^----------------------------------------------
``